lib/commands/general.js

```diff
@@ -44,5 +45,22 @@ commands.getWindowSize = async function (windowHandle = 'current') {
   return await this.executeAtom('get_window_size', []);
 };
 
+commands.getDeviceTime = iosCommands.general.getDeviceTime;
```

Comment:
If that is a method that exists in appium-ios-driver, and is in a file in the `/commands/` tree, then yes. They are exported here: https://github.com/appium/appium-ios-driver/blob/master/lib/commands/index.js#L30-L47